packages/gatsby-theme-docs/gatsby-node.js

```diff
@@ -131,21 +131,31 @@ exports.createPages = async ({ graphql, actions }, options) => {
 
   const sidebarContents = getSidebarContents(sidebarCategories, edges);
 
-  const [owner, repo] = githubRepo.split('/');
+  const [baseSite, owner, repo] = gitRepo.split('/');
   const template = require.resolve('./src/index.js');
 
   edges.forEach(edge => {
     const { id, title, relativePath } = edge.node;
     const { fields } = getPageFromEdge(edge);
 
+    let url;
+
+    if (baseSite.includes('github')) {
```

Comment:
Getting back late to this review. Instead of checking the urls for our own websites it should be generic to where anyone can configure their own site. What I would do is add a new option in `@availity/gatsby-theme-docs` and call it something like `gitType`. From that you can allow either `bitbucket`, `github`, or `gitlab`. That way its configurable for anyone and they still just put their repoUrl in the `gitRepo`  option and we will handle the rest link you are doing in this highlighted snippet.